tests/safeds_stubgen/stubs_generator/__snapshots__/test_generate_stubs.ambr

```diff
@@ -0,0 +1,152 @@
+# serializer version: 1
+# name: test_class_attribute_creation
+  '''
+  package test_stub_generation.attribute_module
+  
+  class A {}
+  
+  class B {
+  	attr type_hint_public: Int
+  	attr no_type_hint_public
+  	attr object_attr: A
+  	// Todo Tuple types are not allowed in SafeDS
+  	attr tuple_attr_1: Tuple
+  	// Todo Tuple types are not allowed in SafeDS
+  	attr tuple_attr_2: Tuple<union<String, Int>>
+  	// Todo Tuple types are not allowed in SafeDS
+  	attr tuple_attr_3: Tuple<String, Int>
+  	attr list_attr_1: List<Any>
+  	attr list_attr_2: List<union<String, A>>
+  	// Todo List type has to many type arguments
+  	attr list_attr_3: List<String, A>
+  	// Todo List type has to many type arguments
+  	attr list_attr_4: List<String, union<A, Int>>
+  	attr dict_attr_1: Map<Any, Any>
+  	attr dict_attr_2: Map<String, Int>
+  	attr dict_attr_3: Map<union<String, Int>, A?>
+  	attr bool_attr: Boolean
+  	attr none_attr: Nothing?
+  	attr flaot_attr: Float
+  	attr int_or_bool_attr: union<Int, Boolean>
+  	attr str_attr_with_none_value: String
+  	attr mulit_attr_1
+  	attr mulit_attr_3
+  	attr mulit_attr_5
+  	attr mulit_attr_6
+  	attr mulit_attr_7
+  	attr mulit_attr_8
+  	attr init_attr: Boolean
+  }
+  
+  '''
+# ---
+# name: test_class_creation
+  '''
+  package test_stub_generation.class_module
+  
+  class A {}
+  
+  class B(a: Int, b: A?) sub A {
+  	fun f()
+  }
+  
+  class C sub A, B {
+  	attr attr_1: Int
+  	attr attr_2: Int
+  
+  	fun f1()
+  }
+  
+  class D {
+  	class E {
+  		class F {		}
+  	}
+  }
+  
+  '''
+# ---
+# name: test_enum_creation
+  '''
+  package test_stub_generation.enum_module
+  
+  from `enum` import Enum
+  from `enum` import IntEnum
+  from `enum` import Enum as _Enum
```

Comment:
Those imports should not be created.

Reply:
So enums are never imported in Safe-DS stubs? Or are duplicates (```as _Enum```) the problem?